credsweeper/app.py

```diff
@@ -336,6 +348,21 @@ def data_scan(self, data_provider: DataContentProvider, depth: int, recursive_li
             except Exception as gzip_exc:
                 logger.error(f"{data_provider.file_path}:{gzip_exc}")
 
+        elif data_provider.is_encoded():
+            decoded_data_provider = DataContentProvider(data=data_provider.decoded,
+                                                        file_path=data_provider.file_path,
+                                                        file_type=data_provider.file_type,
+                                                        info=f"{data_provider.info}|ENCODED")
+            new_limit = recursive_limit_size - len(decoded_data_provider.data)
+            candidates.extend(self.data_scan(decoded_data_provider, depth, new_limit))
+
+        elif data_provider.is_xml():
```

Comment:
`is_xml()` method can be moved to `Util` class too.

Reply:
similar, is_xml() provides full xml parsing.